Samples/Etg.Yams.ARM/README.md

```diff
@@ -0,0 +1,35 @@
+# Yams Console/Windows Service Host #
+
+The idea with this sample project is to be used as a Yams cluster note on a regular Virtual Machine.
+
+## How to run? ##
+
+In order to run it as a console app, call the following command line:
+
+```
+Etg.Yams.Host.exe -deploymentId:MyDeploymentId -updateDomain:MyUpdateDomain -storageAccount:__MYSTORAGEACC__ -storageKey:__MYSTORAGEKEY__ -updateFrequency:5 -applicationRestartCount:2
+```
+
+If you want to install it as a Windows Service add the `-install` parameter to the command-line.
+
+Replace the parameters as follow:
+
+- deploymentId: The Yams Cluster Deployment Id
+- updateDomain: The update domain
+- storageAccount: Azure Storage account name
+- storageKey: Azure Storage account key
+- updateFrequency: regular Yams update frequency (in second)
+- applicationRestartCount: regular Yams parameter
+
+Those parameters can be set on the project property for debug purposes.
+
+## What is next? ##
+
+1. Create a Chocolatey package and publish it to chocolatey.org
+2. Create an Azure Resource Manager (ARM) Template
+3. Put a on-click-deply link on Yams repo so people will be able to deploy a Yams cluster by just click on this link and fill the wizard parameters.
+
+
+More news soon!
+
+[]s 
```

Comment:
Sorry... Old habit... It means `hugs` :)

Will remove on next PR which will update this file no worries